### PR TITLE
Build for release, auto-updater, website

### LINKED ISSUE #4: Website
Single page website design. Menu on left. Sandwich icon on mobile.

Mention not available on Chrome OS. Mention continuation of Web Server for Chrome.

- [x] Remember site meta tags

Home -> GitHub link, Screenshots and description
Download -> Download links, install instructions, changelog, link to build from source docs
Documentation
&bull; Server Options
&bull; App Settings
&bull; Specific feature docs

- [x] TODO: Favicon

### LINKED ISSUE #5: Auto-updater
Make this optional, and ask when first opened before making any network requests.

Add an option to settings "Check for updates"

Show a banner when updates are available. Check a file at simplewebserver.org/versions.json

### LINKED ISSUE #24: Releases
What all platforms are we going to release this on?

I've been doing some testing and we should be able to do everything from linux to macos to windows store apps.

I've been running into a lot of trouble packaging the appx. I packaged it into the appx but I cannot figure out how to sign it.

Your thoughts?

### PR DESCRIPTION
**Pull request by terreng**
- [x] Closes #5 
- [x] IP change UI logic once #57 merged
- [x] Add compression option
- [x] Build (Closes #24 ) - include Linux?
- [x] Install instructions for website (Also build from source instructions)
- [x] remove auto updater option for microsoft store version
- [x] appx build
- [x] Add missing guides for website (~editing config, https~, logs)
- [x] finish coming from wsc page
- [x] add ipv6 option to docs
- [x] add option to disable logging
- [x] figure out why logging doesn't work on macos
- [x] Update documentation files and README 
- [x] Closes #4 
- [x] Fix settings title cutoff on the g
- [x] Update documentation link in the app to go to exact page

### CODE REVIEW

File: WSC.js [hunk 1]
```diff
@@ -20,16 +20,10 @@ WSC.DirectoryEntryHandler = handlers.DirectoryEntryHandler
...
-        WSC.template_data = data
-    })
-})
-main_fs.getByPath('/directory-listing-template-static.html', function(file) {
-    file.text(function(data) {
-        WSC.static_template_data = data
-    })
-})
+const fs = require('fs')
+const path = require('path')
```
Comment (ethanaobrien) [inline WSC.js:24]: the path and fs libraries are defined on global (see index.js)

Comment (terreng) [reply WSC.js]: Good catch, thanks!

**Comment by ethanaobrien** [comment]
@terreng I dont know if you saw or not, but the htaccess config page (https://simplewebserver.org/docs/swsaccess.html#how-to) is a mess. Could you fix this and possibly redo a lot of the documentation, I really am not good at explaining things in words and sounds like a jumble of info half the time, sorry.

**Comment by terreng** [comment]
> @terreng I dont know if you saw or not, but the htaccess config page (https://simplewebserver.org/docs/swsaccess.html#how-to) is a mess. Could you fix this and possibly redo a lot of the documentation, I really am not good at explaining things in words and sounds like a jumble of info half the time, sorry.

Sure thing, I would be happy to make it neater. If you wan to though, it might be good practice to try rewriting some of it yourself. Being able to write good documentation is an important skill to have when writing code.

I haven't had an opportunity to explore .swshtaccess files, custom scripts, or custom request handlers in very much detail yet. I might have new ideas or suggestions for changes for those features.

**Comment by ethanaobrien** [comment]
>If you wan to though, it might be good practice to try rewriting some of it yourself.

Yeah. If you wouldnt mind could you help me learn how to right those? Like you right some and I use that as an example to write the others and you tell me how I did?

>I might have new ideas or suggestions for changes for those features.

If you do let me know. I do really enjoy working on the whole htaccess thing

**Comment by ethanaobrien** [comment]
Hey @terreng quick question.

I actually have to write a bunch of documentation for one of my other projects and I’ve been thinking about how to do the UI. I was wondering, did you use a template? If so could you point me to what you used.

Thanks!

**Comment by terreng** [comment]
> I actually have to write a bunch of documentation for one of my other projects and I’ve been thinking about how to do the UI. I was wondering, did you use a template? If so could you point me to what you used.

Do you mean the website, or the app UI? I used VuePress for the website (to turn all the .md files into a website). However, I would recommend using something simpler like GitHub pages. It's built-in to GitHub, has several theme options, and gives you a URL.

**Comment by ethanaobrien** [comment]
@terreng 
>Do you mean the website, or the app UI? 

The website, sorry should have specified that

>I used VuePress for the website (to turn all the .md files into a website).

Is this what’s in the website folder?

>However, I would recommend using something simpler like GitHub pages. It's built-in to GitHub, has several theme options, and gives you a URL.

We actually already have a website, I just haven’t started on the documentation part (and the UI of the website is a mess except for this one page)

Thanks!

**Comment by terreng** [comment]
> Is this what’s in the website folder?

Yes

There are quite a few projects similar to it, such as readthedocs.org.

**Comment by ethanaobrien** [comment]
Awesome! Thank you

**Pull request merged**